### ISSUE TITLE
DynamicForm Checkbox Element does not save value. 

### ISSUE THREAD

**Issue by rasseru08** [OrchardCMS/Orchard#5637]
The DynamicForm Checkbox Element does not save its value when the form is submitted.   

Steps
1. Create a form with a Checkbox.
2. View the form,  check the checkbox, and press submit.  
3. The form is submitted but the checkbox value is never written to the database.

**Comment by jchenga** [comment]
For a dynamic form to save a checkbox value, you have to specify a value for the checkbox element in the element editor.

![checkbox](https://cloud.githubusercontent.com/assets/619367/9278244/6b2585a2-4264-11e5-8fbb-5e74c2fb8bd9.JPG)

**Comment by sfmskywalker** [comment]
Exactly. I keep forgetting that myself. Maybe we should add a note about it.

**Comment by rasseru08** [comment]
After some experimenting I managed to figure it out.   However,  if you are going to approach it this way rather than a simple binary (Checked/UnChecked),  you should have a value when checked and a value when not checked.     With the current description of Value I thought it would contain (True/False) as the state of the checkbox.
